### ISSUE TITLE
support sbt 0.13.11

### ISSUE THREAD

**Issue by xuwei-k** [heroku/heroku-buildpack-scala#109]
following code does not match `0.13.11`

```shell
sbtVersion=$(expr "$sbtVersionLine" : 'sbt\.version=\(0\.1[1-3]\.[0-9]\(-[a-zA-Z0-9_]*\)*\)$')
```

https://github.com/heroku/heroku-buildpack-scala/blob/20d03d98c33e941b3b2f7bd1ed1a9ff076ce0925/lib/common.sh#L135

**Comment by jkutner** [comment]
Fixed by 7a99c80cd293d4297c747e23d29c6bfdaf35b3e2

Should have a release today

**Comment by jkutner** [comment]
released in v66
